src/utilities/util_verify.c

```diff
@@ -107,6 +118,6 @@ usage(void)
       "verify %s\n",
       progname, usage_prefix,
       "[-d dump_address | dump_blocks | dump_history | dump_layout | "
-      "dump_offsets=#,# | dump_pages] [-S] uri");
+      "dump_offsets=#,# | dump_pages] [-h] [-S] uri");
```

Comment:
Usage statements should have the options in sorted order.

Reply:
I think you should update the `command-line.dox` file usage line as well.